sim/paladin/runes.go

```diff
@@ -9,14 +9,24 @@ import (
 )
 
 func (paladin *Paladin) ApplyRunes() {
-
 	if paladin.HasRune(proto.PaladinRune_RuneWaistEnlightenedJudgements) {
-		paladin.AddStat(stats.SpellHit, float64(core.SpellHitRatingPerHitChance*17.0))
+		paladin.AddStat(stats.SpellHit, 17*core.SpellHitRatingPerHitChance)
 	}
 
 	paladin.registerTheArtOfWar()
 	paladin.registerSheathOfLight()
 	paladin.registerGuardedByTheLight()
+
+	// "RuneHeadFanaticism" is handled in Exorcism, Holy Shock, SoC, and SoR
+	// "RuneHeadWrath" is handled in Exorcism, Holy Shock, Consecration (and Holy Wrath once implemented)
+
+	paladin.registerHammerOfTheRighteous()
+	// "RuneWristImprovedHammerOfWrath" is handled Hammer of Wrath
+	// "RuneWristPurifyingPower" is handled in Exorcism
+}
+
+func (paladin *Paladin) fanaticism() float64 {
+	return core.TernaryFloat64(paladin.HasRune(proto.PaladinRune_RuneHeadFanaticism), 18, 0) * core.CritRatingPerCritChance
```

Comment:
🚫 I have the same issue about the method name here. It's not clear from usage that this is providing a crit value.

If you want too you can add a shared function for grabbing paladin holy spell bonus crit

```
func (paladin *Paladin) bonusHolySpellCritRating() {
  return paladin.holyPowerCritRating() + paladin.fanaticismCritRating()
}
```

Reply:
Same as above, but can change.